compose.yaml

```diff
@@ -70,6 +70,31 @@ services:
           devices:
             - capabilities: [gpu]
 
+  llama2:
```

Comment:
should there be a new profile for when you want to spin up the ETL with this?

Reply:
I wanted to avoid profiles and keep this manual for now, until usage is more obvious...

I agree it probably shouldn't go in the normal support profile yet/ever because the resource profile on this will be so intense. You likely have a custom machine for llama2, unlike normal ETL where you can get away with doing everything locally.

But separately... I'm starting to feel the inflexibility of offloading the responsibility of this docker compose stuff to the user. Like, the other services (ctakes & cnlp) are just in service of one task (so far), and our instructions have you start them regardless of whether you need them at all. As we add more like that, it would be nice if the ETL sat above/next-to the docker compose layer and could also drive it, starting and stopping services.  (that doesn't really jive with the "llama2 on a separate machine" idea above, but just a thought)

Reply:
it might be worth looking into docker swarm? not as part of this, of course, but allows for some dynamism in container spinup.